data/socs.yml

```diff
@@ -2153,3 +2153,138 @@
   - bti
   name: Yitian 710
   vendor: Alibaba
+- cores:
+  - implementer: 0x41
+    part: 0xd03
+    revision: 0x3
+    variant: 0x0
+  features:
+  - fp
+  - asimd
+  - aes
+  - pmull
+  - sha1
+  - sha2
+  - crc32
+  - cpuid
+  name: S5P6818
```

Comment:
I love cryptic names used by vendors. Samsung had chip with same name (or maybe it was the same one).